python/pip_install/pip_repository.bzl

```diff
@@ -513,6 +528,22 @@ python_interpreter. An example value: "@python3_x86_64-unknown-linux-gnu//:pytho
     "repo_prefix": attr.string(
         doc = """
 Prefix for the generated packages will be of the form `@<prefix><sanitized-package-name>//...`
+""",
+    ),
+    "target_platforms": attr.string_list(
```

Comment:
Please add the list of supported values into the doc somewhere. Or have it set as the `values` arg (which enforces the value is in the given list). Otherwise you have to fail first to see the values. Much nicer to have it seen in the docs.